pkg/review/commit_review_status.go

```diff
@@ -61,6 +66,16 @@ WHERE
   AND issues.closed_at >= TIMESTAMP('%s')
 `
 
+// Commit maps the columns from the driving BigQuery query
+// to a usable structure.
+type Commit struct {
+	Author       string `bigquery:"author"`
+	Organization string `bigquery:"organization"`
+	Repository   string `bigquery:"repository"`
+	Sha          string `bigquery:"commit_sha"`
+	Timestamp    string `bigquery:"commit_timestamp"`
```

Comment:
Perhaps consider adding a comment here that explains the format of the timestamp (millis, micros, RFC1234, etc).

Reply:
done.